Intel_SighPP_CMake/Intel_SighPP_CMake/src/ml_lib/caffe_impl.cpp

```diff
@@ -25,7 +26,12 @@ struct CaffeModelImpl : public ModelInterface {
 	const float meanVal = 127.5;
 
 	const float confidence_threshold = 0.8f;
-
+	/// <summary>
+	/// Constrcutor to create a caffe-based MobileNet SSD object recognition network
```

Comment:
Constructor is misspelled 😄 
Also, see comment above, isn't this more general that just MobileNet SSD?